src/main/java/com/direwolf20/buildinggadgets/common/entities/BlockBuildEntity.java

```diff
@@ -21,62 +20,98 @@
 import java.util.Optional;
 
 public class BlockBuildEntity extends EntityBase {
-    private static final DataParameter<Integer> toolMode = EntityDataManager.<Integer>createKey(BlockBuildEntity.class, DataSerializers.VARINT);
+
+    public static enum Mode {
```

Comment:
Question: Should we put enums like this at the bottom of classes as they're a bit messy? @MajorTuvok @Phylogeny

Reply:
I'm used to putting enums (enums only) at the top of classes

Reply:
Then we'll keep it like that